tools/run_cargo_fmt.sh

```diff
@@ -49,15 +51,22 @@ let FAIL=0
 set -e
 
 # Find folders with Cargo.toml files in them and run `cargo fmt`.
-if [ "$1" == "diff" ]; then
-	# Just print out diffs and count errors, used by Travis
-	CARGO_FMT_ARGS="-- --check"
-fi
 for f in $(find . | grep Cargo.toml); do
+	printf "\rFormatting %-$((39))s" $(dirname $f)
```

Comment:
I was skeptical about this line, but it works well. However, I don't think we should add it. Using `cargo fmt --all` instead of this loop is over twice as fast on my computer. I think we should revisit my PR and switch. 2x performance seems worth having to custom case the tools directory.

Reply:
Hah, I implemented this before I saw this comment :)